PowerPointLabs/PowerPointLabs/TooltipsLab/TooltipsLabConstants.cs

```diff
@@ -1,8 +1,50 @@
-﻿
+using System.Drawing;
+
+using Microsoft.Office.Core;
+using Microsoft.Office.Interop.PowerPoint;
+
 namespace PowerPointLabs.TooltipsLab
 {
     static class TooltipsLabConstants
     {
+        public const MsoAutoShapeType TriggerShape = MsoAutoShapeType.msoShapeOval;
+        public const string ShapeNameHeader = "msoShape";
+        public const string AnimationNameHeader = "msoAnimEffect";
+        public const string CalloutNameSubstring = "Callout";
+
+        public static readonly MsoAnimEffect[] AnimationEffects = new MsoAnimEffect[]
+        {
+            MsoAnimEffect.msoAnimEffectAppear,
+            MsoAnimEffect.msoAnimEffectBounce,
+            MsoAnimEffect.msoAnimEffectFade,
+            MsoAnimEffect.msoAnimEffectFloat,
+            MsoAnimEffect.msoAnimEffectFly,
+            MsoAnimEffect.msoAnimEffectGrowAndTurn,
+            MsoAnimEffect.msoAnimEffectRandomBars,
+            MsoAnimEffect.msoAnimEffectPlus,
+            MsoAnimEffect.msoAnimEffectSplit,
+            MsoAnimEffect.msoAnimEffectSwivel,
+            MsoAnimEffect.msoAnimEffectWheel,
+            MsoAnimEffect.msoAnimEffectWipe,
+            MsoAnimEffect.msoAnimEffectZoom
+        };
+        public static readonly Bitmap[] AnimationImages = new Bitmap[]
+        {
+            Properties.Resources.Animation_Appear,
+            Properties.Resources.Animation_Bounce,
+            Properties.Resources.Animation_Fade,
+            Properties.Resources.Animation_Float_In,
+            Properties.Resources.Animation_Fly_In,
+            Properties.Resources.Animation_Grow___Turn,
```

Comment:
Is there any reason for the 3 underscores?

Reply:
This was converted from Grow & Turn. Visual Studio replaces spaces and `&` with `_`, so this is the resulting name. Should this be addressed?

Reply:
Okay if that's the case then it should be fine! Just worried that it was a typo :)